pulp_2_tests/constants.py

```diff
@@ -0,0 +1,548 @@
+# coding=utf-8
+"""Values usable by multiple test modules."""
+
+from urllib.parse import quote_plus, urljoin
+from types import MappingProxyType  # used to form an immutable dictionary
+
+from pulp_smash.constants import PULP_FIXTURES_BASE_URL, PULP_FIXTURES_KEY_ID  # pylint:disable=unused-import
```

Comment:
Why disable the unused import error? Wouldn't it be better to drop the unused import?